homeassistant/components/islamic_prayer_times/sensor.py

```diff
@@ -38,8 +38,8 @@
         translation_key="maghrib",
     ),
     SensorEntityDescription(
-        key="Isha",
-        translation_key="isha",
+        key="Ishaa",
```

Comment:
this is used as a unique_id for the entity. changing it will be a breaking change.